config/jest/environment/puppeteer.mjs

```diff
@@ -19,7 +19,7 @@ class BrowserAutomationEnvironment extends PuppeteerEnvironment {
       delete error.stack
 
       // Ensure test fails
-      process.emit('error', error)
+      process.emit('uncaughtException', error)
```

Comment:
The `tsc` compiler has flagged that the 2nd argument can only be type **Error** when emitting `uncaughtException` or `uncaughtExceptionMonitor`

I was a bit wary about this one due to:

* https://github.com/alphagov/govuk-frontend/pull/2933

It's the `delete error.stack` line above that's vital, as we were unable to do this before Jest Puppeteer's [`exitOnPageError`](https://github.com/alphagov/govuk-frontend/blob/main/jest-puppeteer.config.js#L9) handler fired previously.

Appears to be due to Jest's handling of error stack traces in: https://github.com/facebook/jest/pull/6281/files#diff-ec0ee442213b8202b52930e84121ed8a150142a0a8d8f8edff37ccf5d7029a34R56